spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/ReadIntegrationTestBase.java

```diff
@@ -375,4 +379,112 @@ public void testArrowCompressionCodec() {
 
     assertThat(avroResults).isEqualTo(arrowResultsForLZ4FrameCodec);
   }
+
+  private void uploadFileToGCS(String resourceName, String destinationURI, String contentType) {
+    try {
+      AcceptanceTestUtils.uploadToGcs(
+          getClass().getResourceAsStream("/integration/" + resourceName),
+          destinationURI,
+          contentType);
+    } catch (Exception exception) {
+      exception.printStackTrace();
+    }
+  }
+
+  @Test
+  public void testReadFromBigLakeTable_csv() {
```

Comment:
All ` testReadFromBigLakeTable_*` tests can use the same internal implementation with different parameters. No need for code redundancies